### ISSUE TITLE
PizzaBot

### ISSUE THREAD

**Issue by AlisaBrljak** [AlisaBrljak/Alisa-Brljak#1]
PizzaBot is your personal pizza ordering assistant, built entirely in Python. With just a few lines of code, you can customize your pizza order and get an instant price quote. Whether you're a fan of classic toppings or experimental flavour combinations, PizzaBot has got you covered. And the best part? No waiting on hold or talking to a human required!

**Comment by AlisaBrljak** [comment]
The PizzaBot is a convenient and efficient way to satisfy pizza cravings without any hassle. However, it's important to note that automation should not completely replace human interaction and customer service, as there may be situations where personal attention is necessary.

**Comment by AlisaBrljak** [comment]
My PizzaBot can quickly take and process orders, reducing wait times and improving overall efficiancy. With automation, there is less room for human errors in taking orders, ensuring that every customer receives the same level of service. Customers can place orders at any time of day or night, and from anywhere with an internet connection, making the process much more convenient. This software can help eliminate misunderstandings or errors in orders, resulting in greater customer satisfaction. This system can easily scale up to handle large volumes of orders during peak hours, without the need for additional staff.Do you want a more personalized experience? Try the advanced machine learning algorithm, Also known as, The PizzaBot!
